src/tokens/saga.test.ts

```diff
@@ -381,14 +387,103 @@ describe(fetchImportedTokenBalances, () => {
       [mockTestTokenTokenId]: {
         ...mockImportedTokens[mockTestTokenTokenId],
         balance: new BigNumber(0.000000001).toFixed(),
+        priceUsd: undefined,
       },
       [mockPoofTokenId]: {
         ...mockImportedTokens[mockPoofTokenId],
         balance: new BigNumber(0.0005).toFixed(),
+        priceUsd: undefined,
       },
       [mockUSDCTokenId]: {
         ...mockImportedTokens[mockUSDCTokenId],
         balance: new BigNumber(10).toFixed(),
+        priceUsd: undefined,
+      },
+    })
+  })
+})
+
+describe(fetchImportedTokenBalances, () => {
```

Comment:
is this a bad merge? it seems like this test group already exists?

Reply:
Oh, yes, I messed it up in the merge 🤯